src/App.tsx

```diff
@@ -1,11 +1,87 @@
 import './installSesLockdown';
-import 'ses';
+import { motion } from 'framer-motion';
+import { useEffect } from 'react';
+import { useAtom } from 'jotai';
+import { E, ERef } from '@endo/eventual-send';
+import { makeAsyncIterableFromNotifier as iterateNotifier } from '@agoric/notifier';
+
+import {
+  brandToInfoAtom,
+  walletAtom,
+  pursesAtom,
+  offersAtom,
+} from 'store/store';
+import WalletConnection from 'components/WalletConnection';
+import { INTER_LOGO } from 'assets/assets';
+import Swap from 'components/Swap';
+
+const watchPurses = async (
+  wallet: ERef<any>,
```

Comment:
yay types! we do have a type for this. let's try to import from agoric-sdk. not blocking.

Reply:
The type for this is that of https://github.com/Agoric/agoric-sdk/blob/master/packages/wallet/ui/src/service/ScopedBridge.js#L44. I think I'll need to add a type declarations file to @agoric/wallet/ui to properly use that unless I'm missing something. Right now it only exports "@agoric/wallet/ui/build"